inc/document.class.php

```diff
@@ -1648,4 +1648,66 @@ public static function getImage($path, $context, $mwidth = null, $mheight = null
       );
       return ($result ? $context_path : $path);
    }
+
+   /**
+    * Give cron information
+    *
+    * @param string $name task's name
+    *
+    * @return arrray of information
+   **/
+   static function cronInfo($name) {
+
+      switch ($name) {
+         case 'cleanorphans' :
+            return ['description' => __('Clean orphaned documents')];
+      }
+      return [];
+   }
+
+   /**
+    * Cron for clean orphan documents (without Document_Item)
+    *
+    * @param Crontask $task Crontask object
+    *
+    * @return integer (0 : nothing done - 1 : done)
+   **/
+   static function cronCleanOrphans(Crontask $task) {
```

Comment:
Problem with this is that it will delete all documents added directly from the `/front/document.form.php` form. As it is possible to create documents that are not related to any item, it is pretty sure that this feature is used by some users.

Reply:
well, if they want some documents that are not related to anything, they should just not enable this crontask